projects/components/src/table/cells/data-renderers/string-array/string-array-table-cell-renderer.component.ts

```diff
@@ -51,7 +51,7 @@ export class StringArrayTableCellRendererComponent extends TableCellRendererBase
   public ngOnInit(): void {
     super.ngOnInit();
 
-    this.firstItem = this.value[0] ?? '-';
+    this.firstItem = this.value[0] ?? '';
```

Comment:
if this is null, it should still be `-` right? But better, could we just use display string pipe in the template here?